front/app/components/admin/ContentBuilder/Toolbox/DraggableElement.tsx

```diff
@@ -18,59 +19,90 @@ const StyledBox = styled(Box)`
 interface ToolboxItemProps {
   label: string;
   icon: IconNames;
+  disabled?: boolean;
 }
 
-const ToolboxItem = ({ icon, label }: ToolboxItemProps) => {
-  return (
-    <StyledBox
-      width="100%"
-      display="flex"
-      paddingLeft="10px"
-      alignItems="center"
-    >
-      <Icon
-        marginRight="16px"
-        width="20px"
-        height="20px"
-        fill={colors.primary}
-        name={icon}
-      />
-      <Text color="textPrimary" style={{ lineHeight: '1' }}>
-        {label}
-      </Text>
-    </StyledBox>
-  );
-};
+const ToolboxItem = forwardRef(
+  (
+    { disabled, icon, label }: ToolboxItemProps,
+    ref: React.RefObject<HTMLDivElement>
+  ) => {
+    return (
+      <StyledBox
+        width="100%"
+        display="flex"
+        paddingLeft="10px"
+        alignItems="center"
+        ref={ref}
+      >
+        <Icon
+          marginRight="16px"
+          width="20px"
+          height="20px"
+          fill={disabled ? colors.textSecondary : colors.primary}
+          name={icon}
+        />
+
+        <Text
+          color={disabled ? 'textSecondary' : 'textPrimary'}
+          style={{ lineHeight: '1' }}
+        >
+          {label}
+        </Text>
+      </StyledBox>
+    );
+  }
+);
 
-const DraggableContainer = styled.div`
+const DraggableContainer = styled.div<{ disabled?: boolean }>`
   width: 100%;
-  cursor: move;
+  cursor: ${(props) => (props.disabled ? 'not-allowed' : 'grab')};
 `;
 
 interface Props extends ToolboxItemProps {
   id: string;
   component: React.ReactElement;
+  disabled?: boolean;
+  tooltipContent?: React.ReactNode;
 }
 
-const DraggableElement = ({ id, component, icon, label }: Props) => {
+const DraggableElement = ({
```

Comment:
We need a disabled state for this element - for examples when proposals are not enabled on a platform.